### ISSUE TITLE
Menu Item: ui-state-hover and ui-state-active classes gained after click | v14.0.0+ 

### ISSUE THREAD

**Issue by MarcelHofherr** [primefaces/primefaces#12560]
### Describe the bug

Very Similar to this: [#12378](https://github.com/primefaces/primefaces/issues/12378)

If you hover over a MenuItem to remove the state-active and state-hover classes and then e.g. click on the white space next to the Menu the first MenuItem regains ui-state-focus and ui-state-active classes.

### Reproducer

Go to: https://www.primefaces.org/showcase/ui/menu/menu.xhtml
Hover over MenuItem to remove the Highlighting and Focus
Click on the white space next to it.

### Expected behavior

First item shouldn't regain the style classes. 
(Unless if the whole behaviour is intended by https://www.w3.org/WAI/standards-guidelines/wcag/)

### PrimeFaces edition

Community

### PrimeFaces version

14.0.4

### JSF implementation

Mojarra

### JSF version

2.3

### Java version

11

**Comment by melloware** [comment]
Yes this was all added by WCAG which states that if a menu is displayed its first item should be the active item.

**Comment by melloware** [comment]
Is it causing an issue?

**Comment by melloware** [comment]
Is that a static menu (meaning always on screen) or a Popup Menu.  Maybe I can disable it for Static always displayed menus.

**Comment by MarcelHofherr** [comment]
Yes, it's a static menu

**Comment by melloware** [comment]
Got it I think I can fix this scenario. Will report back.

**Comment by melloware** [comment]
PR submitted.

**Comment by melloware** [comment]
14.0.5 is released please test

**Comment by MarcelHofherr** [comment]
Works! Thanks.
Though one thing i noticed:
If you click e.g. between Menu Items inside the plain menu it doesn't highlight. Once you then hover over the next item the first item in the Menu gets highlighted
